src/OptiHPLCHandler/empower_instrument_method.py

```diff
@@ -46,7 +46,7 @@ def __getitem__(self, key: str) -> str:
         try:
             xml = self.current_method["xml"]
         except KeyError:
-            raise KeyError(f"No xml found in method definition")
```

Comment:
When re-raising an exception the correct syntax is:
```
except Exception as ex:
    raise MyException("More details here") from ex
```
in the end the stack trace ends up with the same information but you will avoid it saying "During handling of the above exception, another exception occurred" - which in my mind indicate a bug.

Reply:
Cool, I did not know that.

If you have suggested changes where you can write the new code, you can make a suggestion that I can accept directly with the icon of a page with +- on it, to the left of heading.